packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -61,7 +61,7 @@ export function Header({
     <div className="border-default relative z-10 flex border-b border-l px-5 py-4">
       <div className="flex items-center gap-3">
         <h3 className="min-w-[150px] text-base font-semibold leading-4">
-          {selectedDate.format("MMM D")}-{selectedDate.add(extraDays, "days").format("D")},{" "}
+          {selectedDate.format("MMM D")}-{selectedDate.add(extraDays - 1, "days").format("D")},{" "}
```

Comment:
extraDays are added including the day of the selection, hence subtracted one from it.